PayloadPkg/FirmwareUpdate/FirmwareUpdateHelper.c

```diff
@@ -1033,6 +1036,73 @@ CheckAcmSvn (
   return EFI_SUCCESS;
 }
 
+#if defined (PLATFORM_RPLA)
```

Comment:
It looks the added code is common for all the platforms.

Reply:
done.